docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Enables alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Enables alert suppression for rules with non-sequence EQL queries ({kibana-pull}176422[#176422]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now GA ({kibana-pull}180838[#180838]).
+* Allows you to add newly created fields to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `discover:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
+* Makes conversations with Elastic AI Assistant persist across sessions ({kibana-pull}173487[#173487]).
+* Adds conversation streaming for Elastic AI Assistant ({kibana-pull}180095[#180095]).
+* Adds support for Anthropic Claude 3 to the Amazon Bedrock connector, and makes it the default model ({kibana-pull}179304[#179304]).
+* Adds an **AI Assistant** settings section to the **Management** menu ({kibana-pull}176656[#176656]).
+* Adds the `_source` field to the Alert details flyout's JSON view ({kibana-pull}180477[#180477]).
+* Improves the UI for row renderers in Timeline ({kibana-pull}180669[#180669]).
+* Allows data collected by AuditBeat to appear in Session View ({kibana-pull}179985[#179985]).
+* Improves the visual appearance of the asset criticality alert column ({kibana-pull}180868[#180868]).
+* Adds an advanced setting that disables alert enrichment via memory scanning for malicious behavior alerts ({kibana-pull}180636[#180636]).
+* Adds an advanced setting that lets you turn off a performance optimization that makes malware on-write and file event processing asynchronous ({kibana-pull}179179[#179179]).
+* Makes the alert details flyout's state persist ({kibana-pull}178746[#178746], {kibana-pull}179511[#179511]).
```

Comment:
@PhilippeOberti would you mind verifying that the small addition I made below is correct? Thank you!

```suggestion
* Makes the alert details flyout's state persist in your browser ({kibana-pull}178746[#178746], {kibana-pull}179511[#179511]).
```

Reply:
this is correct, though you could say _Makes some of the alert and event details flyout's state persist_:
- it's not the entire state that persists
- this would be applied to non-alert flyout as well

Reply:
```suggestion
* Makes some of the flyout's state persist for alert and event details ({kibana-pull}178746[#178746], {kibana-pull}179511[#179511]).
```